src/lib/components/End/OnDate.js

```diff
@@ -13,34 +13,57 @@ const EndOnDate = ({
     date,
     options,
   },
+  calendarComponent: CustomCalendar,
   handleChange,
 }) => {
+  // Check if CustomCalendar has been provided and is different from default blank function
+  const isThereCustomComponent = CustomCalendar.toString() !== 'function calendarComponent() {}';
```

Comment:
I know how strange it looks. It's because airbnb eslint-rules don't allow propTypes without .isRequired. So I provide an empty function by default.